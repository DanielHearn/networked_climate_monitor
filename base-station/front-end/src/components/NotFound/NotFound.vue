<template>
  <div>
    <p>404 - Not Found</p>
  </div>
</template>

<script>
export default {
  name: 'not-found'
}
</script>