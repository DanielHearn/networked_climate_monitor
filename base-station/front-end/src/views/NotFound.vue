<template>
  <div class="home">
    <not-found />
  </div>
</template>

<script>
import NotFound from './../components/NotFound/NotFound.vue'

export default {
  name: 'settings',
  components: {
    NotFound
  }
}
</script>
