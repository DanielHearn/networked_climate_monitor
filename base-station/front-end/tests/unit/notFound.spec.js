import { shallowMount } from '@vue/test-utils'
import NotFound from './../../src/views/NotFound.vue'

describe('NotFound.vue', () => {
  let wrapper

  beforeEach(() => {
    wrapper = shallowMount(NotFound, {
      stubs: ['router-link']
    })
  })

  afterEach(() => {
    wrapper.destroy()
    jest.resetAllMocks()
  })

  it('has the expected html structure', () => {
    expect(wrapper.element).toMatchSnapshot()
  })
})
