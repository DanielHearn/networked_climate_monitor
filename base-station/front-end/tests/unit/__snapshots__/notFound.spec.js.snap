// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound.vue has the expected html structure 1`] = `
<div>
  <p>
    Error 404: Page not found
  </p>
   
  <router-link-stub
    class="link title--nav"
    to="/"
  >
    Click here to go back to the home page
  </router-link-stub>
</div>
`;
